airflow-core/tests/unit/dag_processing/test_manager.py

```diff
@@ -1158,3 +1158,61 @@ def test_bundle_names_to_parse(self, bundle_names, expected, configure_dag_bundl
 
         bundle_names_being_parsed = {b.name for b in manager._dag_bundles}
         assert bundle_names_being_parsed == expected
+
+    @pytest.mark.usefixtures("clear_parse_import_errors")
+    @conf_vars({("core", "load_examples"): "False"})
+    def test_clear_orphaned_import_errors_path_consistency(self, tmp_path, configure_testing_dag_bundle):
```

Comment:
This test doesn't fail on current main and I suspect the issue you encountered has been fixed